src/execution.ts

```diff
@@ -1160,12 +1160,13 @@ function safeMap(
 }
 
 const MAGIC_MINUS_INFINITY =
-  "__MAGIC_MINUS_INFINITY__71d4310a-d4a3-4a05-b1fe-e60779d24998";
+  "__MAGIC_MINUS_INFINITY__71d4310a_d4a3_4a05_b1fe_e60779d24998";
 const MAGIC_PLUS_INFINITY =
-  "__MAGIC_PLUS_INFINITY__bb201c39-3333-4695-b4ad-7f1722e7aa7a";
-const MAGIC_NAN = "__MAGIC_NAN__57f286b9-4c20-487f-b409-79804ddcb4f8";
+  "__MAGIC_PLUS_INFINITY__bb201c39_3333_4695_b4ad_7f1722e7aa7a";
+const MAGIC_NAN = "__MAGIC_NAN__57f286b9_4c20_487f_b409_79804ddcb4f8";
```

Comment:
I replaced the dashes with underscores, since the dashes need to be escaped in a regex sometimes.

Reply:
Sounds good.